app/src/main/java/com/example/king_bob_nae/features/myprofile/BindingAdapter.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.king_bob_nae.features.myprofile
+
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.databinding.BindingAdapter
+import com.example.king_bob_nae.R
+import com.example.king_bob_nae.features.myprofile.domain.userprofile.UserProfileUiState
+
+object BindingAdapter {
+    @JvmStatic
+    @BindingAdapter("bindBookMark")
+    fun ImageView.bindBookMark(item: UserProfileUiState.ScrapedImage) {
+        this.background = if (item.clicked) {
+            ContextCompat.getDrawable(this.context, R.drawable.ic_bookmark_on)
+        } else {
+            ContextCompat.getDrawable(this.context, R.drawable.ic_bookmark_off)
+        }
```

Comment:
```suggestion
        background =  ContextCompat.getDrawable(context, if (item.clicked) R.drawable.ic_bookmark_on
        else R.drawable.ic_bookmark_off)
```
이렇게 바꾸면 좀 더 깔끔할거같아요~